app/lib/server/server.go

```diff
@@ -255,7 +256,7 @@ func checkList(list *sda.Roster, descs []string) error {
 	}
 	log.Lvl3("Sending message to: " + serverStr)
 	msg := "verification"
-	log.Info("Checking server(s) ", serverStr, ": ")
+	fmt.Printf("Checking server(s) %s: ", serverStr)
```

Comment:
What about having a `log.Infof` ? (that does not add a "\n" unless you explicitely put)